src/store/actions/authActions.js

```diff
@@ -0,0 +1,57 @@
+export const register = newUser => {
```

Comment:
all of the auth actions implemented here are just a promises that runs on react-redux-firebase { getFirebase, getFirestore} methods, who sent as extra arguments to the thunk-middlewareat the store.

for more info - [react-redux-firebase github](uhttps://github.com/prescottprue/react-redux-firebaserl)